### ISSUE TITLE
Incompatible signature for Page::createTranslation

### ISSUE THREAD

**Issue by jbcr** [BitBagCommerce/SyliusCmsPlugin#319]
In my Sylius 1.8 project, I use this plugin `bitbag/cms-plugin` version 2.4.0 but I have this error:

```
 Fatal error: Declaration of BitBag\SyliusCmsPlugin\Entity\Page::createTranslation(): ?BitBag\SyliusCmsPlugin\Entity\PageTranslationInterface must be compatible with 
Sylius\Component\Resource\Model\TranslatableTrait::createTranslation(): Sylius\Component\Resource\Model\TranslationInterface in /builds/project/vendor/bitbag/cms-plugin/src/Entity/Page.php on line 177
```

**Comment by kptLucek** [comment]
Hi there!

Does the problem still persists?
What is your PHP, Symfony version?

**Comment by jbcr** [comment]
PHP 7.4 on Alpine ou Debian docker image

**Comment by jbcr** [comment]
@kptLucek This error is also triggered on PHP 8.0 RC4

**Comment by mateusz-kolecki** [comment]
Same for me, PHP 7.4, Sylius 1.7.4, bitbag/cms-plugin v2.4.0

**Comment by Tomsgu** [comment]
Same problem PHP 7.4, Sylius 1.8, bitbag/cms-plugin v3.0.0-beta1.

**Comment by jcgdjob** [comment]
Why still not merging the code?

**Comment by leszczuu** [comment]
fixed with 3.0.0 release
